hack/upgrade-test.sh

```diff
@@ -43,6 +43,18 @@
 # to verify that it is updated to the new operator image from 
 # the local registry.
 
+function cleanup() {
+    rv=$?
+    if [ "x$rv" != "x0" ]; then
```

Comment:
Why `"x$rv" != "x0"` and not `"$rv" != "0"`?  What's the purpose of the x character?

Reply:
That used to be a trick to guard against leading spaces in string comparison. Probably not required when comparing an exit code though.